session/promise/storage_test.go

```diff
@@ -195,17 +201,53 @@ func Test_LoadPaymentInfo_LoadsProperInfo(t *testing.T) {
 					Amount: 300,
 				},
 				UnconsumedAmount: 200,
+				ConsumerID:       consumerID,
 			},
 		},
 	}
 	ms := newMockStorage(&newMock)
 	s := NewStorage(ms)
-	pi := s.LoadPaymentInfo(issuerID)
+	pi := s.LoadPaymentInfo(issuerID, consumerID)
 	assert.Equal(t, uint64(10), pi.LastPromise.SequenceID)
 	assert.Equal(t, uint64(300), pi.LastPromise.Amount)
 	assert.Equal(t, uint64(200), pi.FreeCredit)
 }
 
+func Test_LoadPaymentInfo_WithPreviousConsumersIssuesNewID(t *testing.T) {
+	newMock := map[string][]StoredPromise{
+		getBucketNameFromIssuer(issuerID): {
+			{
+				SequenceID: 10,
+				AddedAt:    timeMock,
```

Comment:
Where is `consumerID` in mocked DB state?

Reply:
the defaults act as a separate consumer, but i've made it more explicit anwyay.